docs/pages/get-started/start-developing.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Start developing
+description: Make your first change to an Expo project and see it live on your device.
+hideTOC: true
+---
+
+import { Terminal, DiffBlock } from '~/ui/components/Snippet';
+import { Collapsible } from '~/ui/components/Collapsible';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Step } from '~/ui/components/Step';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon, GraduationHat02Icon } from '@expo/styleguide-icons';
+
+import { TemplateFeatures } from '~/scenes/get-started/start-developing/TemplateFeatures';
+import { ProjectStructure } from '~/scenes/get-started/start-developing/ProjectStructure';
+
+<Step label="1">
+## Start a development server
+To start the development server, run the following command:
+
+<Terminal cmd={['$ npx expo start']} />
+
+</Step>
+
+<Step label="2">
+## Open the app on your device
+
+After running the command above, you will see a QR code in your terminal. You can scan this QR code to open the app on your device.
+
+If you're using an Android emulator or iOS simulator, you can press <kbd>a</kbd> or <kbd>i</kbd> respectively to open the app.
+
+<Collapsible summary="Having problems?">
+
+First, make sure you are on the same Wi-Fi network on your computer and your device.
+
+If it still doesn't work, it may be due to the router configuration &mdash; this is common for public networks. You can work around this by choosing the **Tunnel** connection type when starting the development server, then scanning the QR code again.
+
+<Terminal cmd={['$ npx expo start --tunnel']} />
+
+> Using the **Tunnel** connection type will make the app reloads considerably slower than on **LAN** or **Local**, so it's best to avoid tunnel when possible. You may want to install an emulator/simulator to speed up development if **Tunnel** is required for accessing your machine from another device on your network.
```

Comment:
```suggestion
> Using the **Tunnel** connection type will make the app reloads considerably slower than on **LAN** or **Local**, so it's best to avoid tunnel when possible. You may want to install and use an emulator or simulator to speed up development if **Tunnel** is required to access your machine from another device on your network.
```